In [67]:
from gym_cas import *
import math

## Opgave 1

Løs følgende ligninger ved “håndregning”

##### Ligning 1:
$$\dfrac{1}{3x} + 1 = \dfrac{2}{5}$$ <br><br>
$$\dfrac{1}{3x} = \dfrac{2}{5} - 1$$ <br><br>
$$\dfrac{1}{3x} = \dfrac{2}{5} - 1 = \dfrac{2}{5} - \dfrac{5}{5} = -\dfrac{3}{5}$$

##### Ligning 2:
$$\dfrac{10}{3} - \dfrac{2}{4x} = \dfrac{2}{5}$$<br><br>
$$\dfrac{10}{3} = \dfrac{2}{5} + \dfrac{2}{4x}$$<br><br>
$$\dfrac{10}{3} \cdot 4x = \dfrac{2}{5} + \dfrac{2}{4x} \cdot 4x$$<br><br>
$$\dfrac{10}{3} \cdot 4x = \dfrac{2}{5} + 2$$
$$\dfrac{40x}{3} = \dfrac{2}{5} + 2$$<br><br>
$$13,33x = 2,4$$ <br><br>
$$x = 2,4 \div 13,33$$ <br><br>
$$x = 0,18$$

##### Ligning 3:
$$\dfrac{10-x}{3} - \dfrac{2x + 20}{5} = \dfrac{2x}{7}$$ <br> <br>
$$\dfrac{(10-x) \cdot 5}{3 \cdot 5} - \dfrac{(2x + 20) \cdot 3}{5 \cdot 3} = \dfrac{2x}{7}$$ <br> <br>
$$\dfrac{50-5x}{15} - \dfrac{6x + 60}{15} = \dfrac{2x}{7}$$ <br> <br>
$$\dfrac{50-5x - 6x + 60 }{15} = \dfrac{2x}{7}$$ <br> <br>
$$\dfrac{-10 + x}{15} = \dfrac{2x}{7}$$ <br> <br>
$$\dfrac{-10 + x}{15} \cdot 15 \cdot 7 = \dfrac{2x}{7} \cdot 15 \cdot 7$$ <br> <br>
$$-70 + 7x= 30x$$ <br> <br>
$$-70 = 23x$$ <br> <br>
$$\dfrac{-70}{23} = x$$ <br> <br>
$$x = -3,04$$


## Opgave 2

##### Dragejægeren Lilina er 20 år yngre end ridderen Krox. For 3 år siden var Lilina halvt så gammel som Krox.
 • Hvor gammel er dragejægeren Lilina?


$L = K - 20$ <br>
$L - 3 = \dfrac{1}{2} (K - 3)$

Hvis hun skal være fra 3 år siden halvt så gammel som Krox og 20 år yngre så må hun være 23 år


## Opgave 3
Dragejægeren Lilina skal klatre op ad en lodret borgmur. Hun har kastet sit 25 meter lange klatrereb op og
fastgjort det i toppen (vha. sit kasteanker). Når hun spænder klatrerebet helt ud ved at gå væk fra muren og
derefter holder rebet ned til jorden danner det en vinkel på 75∘ med vandret.

• Hvor høj er borgmuren?


For at finde højden af borgmuren benytter vi os af Sinus.

Formel for sinus: $Sin(V)=\dfrac{modstående katete}{hypotenusen}$

In [68]:
Sin(75) * 25

24.1481456572267

Udregningen viser at borgmuren er 24.148 meter høj.

## Opgave 4
Dragejægeren Lilina er kommet op på toppen af muren og lister sig videre til den modsatte borgmur hvor der
er 30 meter lodret ned til det vand der er på denne side af borgen. Lilina skuer ud over vandet, som strækker
sig så langt øjet rækker.

• Hvor langt væk befinder horisonten sig hvis planeten Lilina befinder sig på har en omkreds på 40.000 km?


<br>

For at vi kan finde ud af hvor horisonten er bruger jeg formel: <br>
$horisont = \sqrt{højde \cdot diameter + højde}$

In [69]:
svar = 30 * 40000 / math.pi + 30
sqrt(svar)

618.062993084482

Udregningen her viser at Lilina kan se til horisonten som er 618.062 meter væk

## Opgave 6
Lilina har kæmpet sig ned til borgens køkken hvor hun er ved at blive overmandet af de mange vrede vagter.
Hun bliver nødt til at glide ned ad slisken med køkkenaffald som danner en vinkel på 32∘ med vandret.

• Hvor langt er Lilina gledet når hun er faldet 25 cm i højden?

For at finde ud af hvor langt Lilina er gledet ned af slisken skal vi bruge tangens.


Formel for tangens:   $tan(V)=\dfrac{modstående}{tilstødende}$

$tan(32)=\dfrac{25}{længde}$ <br> <br>
$længde = \dfrac{25}{tan(32)}$

In [70]:
Tan(32)

0.624869351909328

$længde = \dfrac{25}{0,624}$

In [71]:
25/0.624

40.06410256410256

Så Lilina gled cirka 40cm ned af slisken

## Opgave 7

I den vilkårlige trekant ABC kendes siderne 𝑎 = 4,72 og 𝑐 = 7,35 samt vinkel 𝐵 = 56,8∘. <br>
    1. Beregn den resterende side og de resterende vinkler.

I den vilkårlige trekant DEF kendes siden 𝑑 = 4,72 samt vinklerne 𝐸 = 56,8∘ og 𝐹 = 30∘.
    2. Beregn de resterende sider og vinkel.

I den vilkårlige trekant GHI kendes siderne 𝑔 = 5 og ℎ = 4.
    3. Hvis vinklen 𝐻 = 45∘, hvad kan vinklen 𝐼 så være?
    4. Er det en trekant hvis 𝐻 = 60∘?
    5. Hvad er den største værdi som 𝐻 kan være? Hvad er vinkel 𝐺 i dette tilfælde?



1. Beregn den resterende side og de resterende vinkler. <br>
For at finde siden b bruger jeg pythagoras læresætning som fortæller os at $a^2 + b^2 = c^2$

In [72]:
4.72**2 + 7.35**2

76.30089999999998

In [73]:
sqrt(76.30089999999998)

8.73503863757911

Efter viste udregninger ved vi at siden b = 8.73503863757911

Så finder vi siden b vha. cosinusrelationer: $\sqrt{a^2 + c^2 - 2 \cdot a \cdot c \cdot cos(B)}$

In [74]:
sqrt(4.72**2+7.35**2-2*4.72*7.35*Cos(56.8))

6.18940815435473

Så siden b = 6.189

For så at finde vinkel A bruger vi igen cosinusrelationer: $cos(A)=\dfrac{b^2 + c^2 - a^2}{2bc}$

In [75]:
aCos((6.189408**2+7.35**2-4.72**2)/(2*6.189408*7.35))

39.6510830254133

Vinkel A er så 39.651 grader

Vinkel C finder vi på baugrund af vores viden om at der er 180 grader i en trekant.

In [76]:
180 - 39.6510830254133 - 56.8

83.5489169745867

Derfor er vinkel C 83.5489169745867 grader
<br>
<br>

2. Beregn de resterende sider og vinkel.

Vi starter med at finde den sidste vinkel. D.
Det gør vi igen fordi vi ved der er 180 grader i en trekant. 

In [77]:
180 - 56.8 - 30

93.2

Nu har vi alle vinklerne i vores trekant og vi kan begynde at beregne de sidste sider.

nu regener vi siden e

In [78]:
4.72*Sin(93.2)/Sin(30)

9.42528081387790

Siden er er derfor 9.4252808138779

Så regner vi den sidste side som er siden f

In [79]:
4.72*Sin(56.8)/Sin(30)

7.89905511905260

Siden f er derved 7.8990551190526


3. Hvis vinklen 𝐻 = 45∘, hvad kan vinklen 𝐼 så være?

Vi finder siden I med cosinusrelationerne igen: $cos(A)=\dfrac{b^2 + c^2 - a^2}{2bc}$

In [80]:
aCos((4**2+5**2-5.406363**2)/(2*4*5))

72.8855733293959

Vinkel også være: 

In [81]:
aCos((4**2+5**2-1.664705**2)/(2*4*5))

17.1144297194546

Så vinkel I kan være 72.885 grader og 17.114 grader